inst/qml/RegressionLinear.qml

```diff
@@ -63,7 +63,8 @@ Form
 			availableVariablesList.source: ['covariates', 'factors']
 			startIndex:			0
 			availableVariablesListName: "availableTerms"
-			allowedColumns:		[]
+			allowedColumns:		['scale', 'nominal']
```

Comment:
Why did you add these specifications? The column type specification already happens in the main menu, where the user specifies either continuous (covariates) or factors (categorical) predictors, and those types are then respected in the model box. Allowing the type to change seems a bit weird, also since it does not get changed in all models and therefore causing non-nested model comparisons.

Reply:
This change is a vestige of an attempt to fix the variable name encoding problems that I ended up working around with the routines defined in "R/commonClassicalRegression.R". That attempt didn't work, but I forgot to undo this change.